src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala

```diff
@@ -228,8 +233,7 @@ class ManagedGroupRoutesSpec extends FlatSpec with Matchers with ScalatestRouteT
   }
 
   it should "fail with 404 when the group does not exist" in {
-    val samRoutes = TestSamRoutes(resourceTypes)
-
+    val samRoutes = createResource(resourceTypes, Resource(ManagedGroupService.managedGroupTypeName, ResourceId("foo"), Set.empty))
```

Comment:
I think all changes in these 2 files are due to https://github.com/broadinstitute/sam/pull/221#discussion_r224230593, which can't be avoided since this PR is to add auth domain check, which means we need to load resource for the policy we're checking, but previously in tests, we didn't care about the resource that the policy belongs to.

I don't think it'll break backward compatibility since in real use case, we definitely have resource pre-created before we have policy.